.github/pull_request_template.md

```diff
@@ -0,0 +1,67 @@
+This PR resolves Issue #
+
+# Change Summary
+
+- Added component \__a_
+- Updated feature \__b_ to do \__c_ when \__d_
+
+## I Had Difficulty With
+
+- \__problem_
+
```

Comment:
@dargaCode what would you think about keeping just the top two sections just to keep things simple for now?

Reply:
@angelkbrown yeah sounds good! I wanted to create an example with some of the things people might encounter but it's definitely major overkill here =D